middleware/integration_test/bbb-cmd.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+set -eu
+
+# MOCK DEV SCRIPT
+# always return without doing anything
+
+# BitBox Base: system commands repository
+#
+
+# print usage information for script
+usage() {
+  echo "BitBox Base: system commands repository
+usage: bbb-cmd.sh [--version] [--help] <command>
+
+possible commands:
+  setup         <datadir|hostname_link>
```

Comment:
nit: `hostname_link` is included in the help text, but not in the code (and not in the proper `bbb-cmd.sh`)